src/components/Filters/Indicators.tsx

```diff
@@ -1,9 +1,26 @@
-import React, { Component } from 'react';
-import { Select } from 'antd';
-import { STATS, styles } from '../../utils/constants.js';
+import React, {Component} from 'react';
```

Comment:
Is this a stylistic preference? I thought the standard was `import { Component }` like [MDN has it](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import).

Reply:
nope. Needed to set typescript rules. Webstorm treats Typescript as a separate language from JS.